### ISSUE TITLE
Incorrect output of LayerNormalization

### ISSUE THREAD

**Issue by HotzingTone** [tensorflow/tensorflow#58146]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug

### Source

source

### Tensorflow Version

2.10.0

### Custom Code

No

### OS Platform and Distribution

MacOS 12.2.1

### Python version

3.10

### Current Behaviour?

```shell
LayerNormalization doesn't output correct results for certain inputs.
```

### Standalone code to reproduce the issue

```shell
script:
 x = tf.constant([0.09484422, 0.16441762])
 layer = tf.keras.layers.LayerNormalization()
 layer(x)

output:
 <tf.Tensor: shape=(2,), dtype=float32, numpy=array([-0.73995596,  0.73995596], dtype=float32)>

This result is incorrect, it should be close to [-1., 1.].
```

### Relevant log output

_No response_</details>

**Comment by ayushthe1** [comment]
hey ,i just checked it. The output is actually correct.

**Comment by tilakrayal** [comment]
@HotzingTone,
**tf.keras.layers.LayerNormalization** try to Normalize the activations of the previous layer for each given example in a batch independently, rather than across a batch like Batch Normalization. i.e. applies a transformation that maintains the mean activation within each example close to 0 and the activation standard deviation close to 1.

Given a tensor inputs, moments are calculated and normalization is performed across the axes specified in the axis. Please take a look at this official doc [link](https://www.tensorflow.org/api_docs/python/tf/keras/layers/LayerNormalization) for the reference and the gist of it [here](https://colab.research.google.com/gist/tilakrayal/98cc630f1759e6bb9df14cd3b852c687/untitled699.ipynb)

Code:
```

import numpy as np
data = tf.constant(np.arange(10).reshape(5, 2) * 10, dtype=tf.float32)
print(data)

```
Output:

```
tf.Tensor(
[[ 0. 10.]
 [20. 30.]
 [40. 50.]
 [60. 70.]
 [80. 90.]], shape=(5, 2), dtype=float32)

```

Code:
```
layer = tf.keras.layers.LayerNormalization(axis=1)
output = layer(data)
print(output)
```

Output:
```
tf.Tensor(
[[-0.99998    0.99998  ]
 [-0.99998    0.99998  ]
 [-0.99998    0.99998  ]
 [-0.99998    0.99998  ]
 [-0.9999809  0.999979 ]], shape=(5, 2), dtype=float32)
```

**Comment by HotzingTone** [comment]
Hello, could you help me understand why the resulting standard deviation is not close to 1 as the document says it should be?

Input:
```
import tensorflow as tf
x = tf.constant([0.1, 0.2])
layer = tf.keras.layers.LayerNormalization()
layer(x).numpy()
```
Output:
```
array([ 0.84515435, -0.8451543 ], dtype=float32)
```

**Comment by tilakrayal** [comment]
@HotzingTone,
Thanks for opening this issue. Development of keras moved to another [repository](https://github.com/keras-team/keras/issues). 
 
 
Could you please post this issue on keras-team/keras [repo](https://github.com/keras-team/keras/issues).
To know more please refer:
https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999
Thank you!
